hxd/SceneEvents.hx

```diff
@@ -264,16 +299,22 @@ class SceneEvents {
 					}
 				case EOver:
 					isOut = false;
+					selectCursor();
 					continue;
 				case EOut:
 					// leave window
 					isOut = true;
-					if( currentOver != null ) {
+					var i = overList.length - 1;
+					while ( i >= 0 ) {
 						onOut.cancel = false;
-						currentOver.handleEvent(onOut);
-						if( !onOut.cancel )
-							currentOver = null;
+						overList[i].handleEvent(onOut);
+						if ( !onOut.cancel ) {
+							overList.remove(overList[i]);
+							continue;
+						}
+						i--;
 					}
+					selectCursor();

```

Comment:
only if we had an Over that had a cursor set ?

Reply:
Added check to do it only there was overs present. Still need to call it even if no overs left after cycle, as it would reset cursor to Default in that case.